script/Arcueid/Day 8/08_00_ARC08_8_1B.txt

```diff
@@ -85,7 +85,7 @@ This is a lot to take in.
 [sha:8e843741b7b900306b4437a46fe5b50883114e3b]{
 -- Page 549, Offset 21155.
 -- 私たちのこと、あの吸血鬼に聞いてないの？」
-Didn't you hear about us from that vampire of yours?"
+Didn't you hear about us from your little vampire?"
```

Comment:
Not sure how either of these got the possessive/diminutive tbh

Reply:
Just Noel flavor